src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,171 @@
+
+{{site.data.alerts.callout_success}}
+Note that:
+- Clusters with no license key have a 7-day grace period before requiring a license key. This is useful for ephemeral development clusters.
+- No license key is required for developers running [single-node clusters](#single-node-clusters).
+{{site.data.alerts.end}}
+
+## Obtain a license
+
+To obtain a paid **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
+
+To obtain an **Enterprise Free** or **Enterprise Trial** license, take the following steps:
+
+1. Point your web browser to the [CockroachDB {{ site.data.products.cloud }} Console][cloud-console] and [log in]({% link cockroachcloud/authentication.md  %}#console-ui-authentication) as an account with **Organization Admin** permissions.
+1. Select **Organization &raquo; Enterprise Licenses** from the nav bar dropdown. This will bring you to the **Enterprise Licenses** page, which shows a (possibly empty) list of licenses and their keys, with information about the status of each.
+1. Click the **Create License** button. This will bring you to a page called **Get started with CockroachDB Enterprise**.
+1. On this page, you will create an **Enterprise Trial** license or an **Enterprise Free** license.
+    1. To create an **Enterprise Trial** license:
+        1. Fill in the form with the required information.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+    1. To create an **Enterprise Free** license:
+        1. Fill in the form with the required information.
+        1. Toggle the switch called **Find out if my company qualifies for an Enterprise Free license**.
+        1. Fill out the additional information.
+        1. Click the **Continue** button.
+           - Note that at this stage, you may not qualify for the **Enterprise Free** license. If so, you will be given an **Enterprise Trial** license.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+
+{{site.data.alerts.callout_danger}}
+You will not be able to create more than one **Enterprise Trial** license per day. If you try, the UI will prevent you from proceeding, and the following message will be displayed:
+
+> A new Enterprise Trial license cannot be created at this time because one was recently created. Tell us more about your intended use to find out if you qualify for an Enterprise Free license, or create a new one after ${DATE}.
```

Comment:
This block quote isn't working within the callout, so it reads like normal documentation. Maybe italicize it instead?

Reply:
went with italics

refer to:  ARRRRRGGHHHHHHH MARKDOWNNNNNNNNNNNNNNNN